### PR TITLE
Update from 2020 to 2021

### PR DESCRIPTION
**Pull request by Iscaraca**
Additions:
- Reaction roles
- Support for new committees
- A somewhat better verification system

Deletions:
- Redundant code
- Support for committees that no longer exist

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,254 +1,380 @@
...
+ * Workshop (June Event) 698841625627197441
+ * Workshop (March Event) 803645952866385940
+ * Workshop (FOSSASIA) 803646041906872370
+ * Quality Assurance 803646515025281034
+ * Emcees 803646503365246986
+ * Resource Management 803646523681406977
+ * Competition 803646520297521172
+ * Entertainment and Lucky Draw 698841626348486737
+ * Games 698841627606777868
+ * Website and Media 698841629867507752
```
Comment (joelleoqiyi) [inline index.js:108]: Are there any security concern for publicly posting these?

File: index.js [hunk 2]
```diff
@@ -1,254 +1,380 @@
...
+const { ReactionRole } = require("reaction-role");
+const dotenv = require("dotenv");
 
-/* requirements to read bot token as environmental variable */
-const dotenv = require('dotenv'),
-      path   = require('path')
-dotenv.config({path: path.join(__dirname, '../.env')})
+dotenv.config();
+botSecretToken = process.env.BOT_TOKEN;
+botTwoSecretToken = process.env.BOT_TOKEN_TWO;
```
Comment (joelleoqiyi) [inline index.js:8]: Can you create a README.md file or some documentation for how are other developers to know how to set up the secrets, etc.

**Comment by joelleoqiyi** [comment]
Can you also add in other coding workflow documentation. for eg: how to test out the bot, how to set up the secrets tokens stuff, etc etc etc

File: index.js (hunk 1 above)
Comment (Iscaraca) [reply index.js:108]: these role IDs dont unlock any special admin powers so its fine to leave them up for future bot maintainers to refer to.

Comment (joelleoqiyi) [reply index.js:108]: ok! 👍

File: index.js (hunk 2 above)
Comment (Iscaraca) [reply index.js:8]: ive added the readme, do give it a once over thanks

File: README.md [hunk 3]
```diff
@@ -1,4 +1,47 @@
...
+
+## 🎈 Usage <a name = "usage"></a>
+
+To use the reaction role feature, first type &poll in the channel you want to create the message in. Copy the message ID by right-clicking and type &role {messageID} without the curly braces for the reactions.
+
+Current service year: 2021
+
+## 🚀 Deploying your own bot <a name = "deployment"></a>
+
+Ensure that your environment variables using Node.js are pointing to your bot tokens. Push the project to Heroku as a worker (not web).
```
Comment (joelleoqiyi) [inline README.md]: This bot token thing is for individuals to push out their bot right? 

The main question, is how do people test out the bot in the BBCS discord server? for eg: after they make their edits, how do they know whether their code works, and how do they push the bot to the heroku which has the actual BBCS discord bot.

**Review by joelleoqiyi — APPROVED** [review]

**Pull request merged**